### PR TITLE
8281049: man page update for jstatd Security Manager dependency removal

### PR DESCRIPTION
**Pull request by kevinjwalls**
Policy settings are no longer required to run jstatd.

This PR is the change to the source file src/jdk.jstatd/share/man/jstatd.1 and in case that's hard to read, here is a diff of the text as rendered by the man command: https://cr.openjdk.java.net/~kevinw/8281049/jstatd.man.diff.txt

And a copy of the new text rendered by man:
https://cr.openjdk.java.net/~kevinw/8281049/jstatd.man.new.txt

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8281049](https://bugs.openjdk.java.net/browse/JDK-8281049): man page update for jstatd Security Manager dependency removal

### Reviewers
 * [Chris Plummer](https://openjdk.java.net/census#cjplummer) (@plummercj - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.java.net/jdk pull/7320/head:pull/7320` \
`$ git checkout pull/7320`

Update a local copy of the PR: \
`$ git checkout pull/7320` \
`$ git pull https://git.openjdk.java.net/jdk pull/7320/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 7320`

View PR using the GUI difftool: \
`$ git pr show -t 7320`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.java.net/jdk/pull/7320.diff">https://git.openjdk.java.net/jdk/pull/7320.diff</a>

</details>

### CODE REVIEW

File: src/jdk.jstatd/share/man/jstatd.1 [hunk 1]
```diff
@@ -120,15 +120,12 @@ local security policies should be considered before you start the
 \f[CB]jstatd\f[R] process, particularly in production environments or on
 networks that aren\[aq]t secure.
 .PP
-The \f[CB]jstatd\f[R] server installs an instance of
-\f[CB]RMISecurityPolicy\f[R] when no other security manager is installed,
-and therefore, requires a security policy file to be specified.
-The policy file must conform to Default Policy Implementation and Policy
-File Syntax.
+As RMI is in use, the jstatd server uses an ObjectInputFilter to allow
+only essential classes to be deserialized.
```
Comment (plummercj) [inline src/jdk.jstatd/share/man/jstatd.1:124]: How about:
```
For security purposes, the jstatd server uses an RMI ObjectInputFilter to allow only essential classes to be deserialized.
```

Comment (kevinjwalls) [reply src/jdk.jstatd/share/man/jstatd.1]: Yes, sounds good - updated.

File: src/jdk.jstatd/share/man/jstatd.1 [hunk 2]
```diff
@@ -120,15 +120,12 @@ local security policies should be considered before you start the
 \f[CB]jstatd\f[R] process, particularly in production environments or on
 networks that aren\[aq]t secure.
 .PP
-The \f[CB]jstatd\f[R] server installs an instance of
-\f[CB]RMISecurityPolicy\f[R] when no other security manager is installed,
-and therefore, requires a security policy file to be specified.
-The policy file must conform to Default Policy Implementation and Policy
-File Syntax.
+For security purposes, the jstatd server uses an RMI ObjectInputFilter
+to allow only essential classes to be deserialized.
```
Comment (plummercj) [inline src/jdk.jstatd/share/man/jstatd.1:124]: It looks like some formatting is now missing around `jstatd`.

Comment (kevinjwalls) [reply src/jdk.jstatd/share/man/jstatd.1]: Right, missed the quotes we use around jstatd in other places.  Added them in this update - I now see the word **jstatd** in bold in the man output like the other places.

**Review by plummercj — APPROVED** [review]

**Pull request closed without merging**